src/input/user_input.rs

```diff
@@ -807,6 +808,7 @@ where
             }
             Message::ChooseFile(my_index_clone.clone(), result, input_data_clone.clone())
         });
+        let extensions = extensions.join(",");
```

Comment:
@sophie-cluml @jinjoolee07

I think we need to validate each value of `extensions` based on the criteria that has been specified.

Reply:
I was wondering if there’s an existing rule or guideline for validating extensions that I should follow, or would you prefer that I define the criteria? If possible, could you provide more details to help ensure alignment with expectations?